### ISSUE TITLE
why the gallery feature size is [11310,2048]

### ISSUE THREAD

**Issue by LiangHann** [guxinqian/AP3D#2]
For MARS dataset, the gallery has 9330 tracklet, but why get [11310,2048] matrix for gallery set?

**Comment by guxinqian** [comment]
For MARS, gallery set must contain query set, otherwise 140 query imgs will not have ground truth.

**Comment by abhishekaich27** [comment]
If gallery set will have all the query imgs, won't that make the model find all the queries in the gallery rendering test accuracy close to ~100%? Pardon me, if I am missing something.

**Comment by guxinqian** [comment]
In the 47th line of eval_metrics.py, all gallery samples that have the same PID and CamID with query sample are removed. So the test accuracy will not be close to 100%. Adding query set to gallery set is the official evaluation strategy in https://github.com/liangzheng06/MARS-evaluation, otherwise 140 query samples will not have ground truth.

**Comment by abhishekaich27** [comment]
Thank you for the clarification. 

Will it be possible for you to share the I3D pre-trained model weights (Table 5) which has mAP of  83.0%?

**Comment by guxinqian** [comment]
Sorry, due to transferring the server, all pre-trained models are missing. Training the model on 2 GPUs just needs about 12 hours. You can train your own model.
